service/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 springfox.documentation.swagger.v2.path=/api-docs
```

Comment:
Seems like this file was attached to the commit erroneously - suggest removal of this change.

Reply:
I removed these local updates fr my configuration, thanks.